content/guides/hosting/iam/access-management/_index.md

```diff
@@ -30,6 +29,15 @@ Once an organization administrator creates a team, the instance administrator or
 
 Both the organization administrator and the team administrator use team dashboards at `https://wandb.ai/<your-team-name>` to manage teams. For more information, and to configure a team's default privacy settings, see [Add and manage teams]({{< relref "./manage-organization.md#add-and-manage-teams" >}}).
 
+## Maintain admin access
+You must ensure that at least one admin user exists in your instance or organization at all times. Otherwise, no user will be able to configure or maintain W&B.
```

Comment:
```suggestion
You must ensure that at least one admin user exists in your instance or organization at all times. Otherwise, no user will be able to configure or maintain your organization's W&B account.
```